.buildkite/pipeline.yaml

```
@@ -3,30 +3,25 @@ env:
   IMAGE_REPO: ghcr.io/theopenlane/${APP_NAME}
   SONAR_HOST: "https://sonarcloud.io"
 steps:
-  - group: ":test_tube: tests"
+  - group: ":test_tube: Tests"
     key: "tests"
     steps:
       - label: ":golangci-lint: lint :lint-roller:"
-        cancel_on_build_failing: true
         key: "lint"
+        cancel_on_build_failing: true
         plugins:
           - docker#v5.11.0:
-              image: "registry.hub.docker.com/golangci/golangci-lint:v1.60.3"
-              command: ["golangci-lint", "run", "-v", "--timeout", "10m", "--config", ".golangci.yaml", "--concurrency", "0"]
+              image: "ghcr.io/theopenlane/build-image:latest"
+              always-pull: true
+              command: ["task", "go:lint"]
               environment:
                 - "GOTOOLCHAIN=auto"
-      - label: ":golang: go test"
-        key: "go_test"
-        retry:
-          automatic:
-            - exit_status: "*"
-              limit: 2
-        cancel_on_build_failing: true
-        plugins:
           - docker#v5.11.0:
```

This is missing the label and key for the go test: 

```suggestion
      - label: ":golang: go test"
        key: "go_test"
        cancel_on_build_failing: true
        plugins:
          - docker#v5.11.0:
```